pkg/proc/native/proc_windows.go

```diff
@@ -300,6 +301,10 @@ func (dbp *Process) waitForDebugEvent(flags waitForDebugEventFlags) (threadID, e
 		case _RIP_EVENT:
 			break
 		case _EXCEPTION_DEBUG_EVENT:
+			if flags&waitContinueAllExceptions != 0 {
+				continueStatus = _DBG_CONTINUE
+				break
```

Comment:
Are you sure you want to exit from waitForDebugEvent? I would continue with the loop until you see EXIT_PROCESS_DEBUG_EVENT.